### PR TITLE
feat. build frontend cluster image 

### PR DESCRIPTION
**Pull request by lingdie**
### Summary
🚀🌐🛠️

This change adds the Kubernetes resources for deploying the `deploy-manager-frontend` component of the sealos project. This component is a web-based interface for managing Kubernetes clusters using sealos.

> _`sealos` frontend_
> _deployed with ConfigMap, Service_
> _autumn leaves cluster_

### Walkthrough
*  Add a ConfigMap, a Deployment and a Service for the deploy-manager-frontend component ([link](https://github.com/labring/sealos/pull/3016/files?diff=unified&w=0#diff-23b4f3675b37f7ca2bcc3461ac88ebf0a7ea5051c7c269537d39096b7c51d1f8L1-R67))

### CODE REVIEW

**Comment by lingdie** [comment]
@cuisongliu Can you review this and merge it if it's appropriate.

**Comment by cuisongliu** [comment]
Set image publish using gh cmd

File: .github/workflows/frontend-providers.yml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+      - name: Manifest Cluster Images
+        run: |
+          sudo sealos images
+          bash docker/patch/manifest-cluster-images.sh "sealos-cloud-${{ matrix.module }}-frontend"
+        env:
+          OWNER: ${{ github.repository_owner }}
+
+      - name: Sync Images
+        uses: peter-evans/create-or-update-comment@v1
+        if: ${{ github.repository_owner == env.DEFAULT_OWNER }}
```
Comment (cuisongliu) [inline .github/workflows/frontend-providers.yml:144]: DEFAULT_OWNER need set to this workflow

Comment (lingdie) [reply .github/workflows/frontend-providers.yml:144]: done.

**Review by cuisongliu — APPROVED** [review]

**Pull request merged**